data/en/faq.yaml

```diff
@@ -1,6 +1,6 @@
-- title: What is runtime security and why is Falco a solution?
+- title: What is runtime security and how does Falco help?
   content: |
-    Runtime security helps protect workloads against a variety of threats, such as:
+    Runtime security is the process of providing protection for your host, containers, and applications while they’re running against a variety of threats, such as:
```

Comment:
Banned word protection, change to:


Runtime security is the process of providing real-time monitoring or observability capabilities for your host, containers, and applications while they're running. This allows you to detect a variety of threats, such as: